lib/api/src/grpc/proto/qdrant_internal_service.proto

```diff
@@ -7,10 +7,25 @@ service QdrantInternal {
   Get HTTP port for remote host.
    */
   rpc GetHttpPort (HttpPortRequest) returns (HttpPortResponse) {}
+
+  /*
+  Wait until the target node reached the given commit ID.
+   */
+  rpc WaitOnConsensusCommit (WaitOnConsensusCommitRequest) returns (WaitOnConsensusCommitResponse) {}
 }
 
 message HttpPortRequest {}
 
 message HttpPortResponse {
   int32 port = 1;
 }
+
+message WaitOnConsensusCommitRequest {
+  int64 commit = 1; // Raft commit as u64
+  int64 term = 2; // Raft term as u64
+  int64 timeout = 3; // Timeout in seconds
```

Comment:
I guess "Timeout in seconds" works here, but I'd prefer
- either `timeout_secs` instead of `timeout`
- or, better, `timeout_ms` (in milliseconds)

Reply:
I agree that it would be better. However, we have 5 other gRPC messages where `int64 timeout` is used. This tries to keep it consistent.